troposphere/static/js/components/settings/AdvancedSettingsPage.jsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
+
 import SSHConfiguration from "components/settings/advanced/SSHConfiguration";
+import ClientCredentials from "components/settings/advanced/ClientCredentials";
```

Comment:
The name here is ClientCredentials, yet above the file referenced is ExportCredentials?